### ISSUE TITLE
Encoding IplImage.ToImage() and saving to file, using go image, creates a black image

### ISSUE THREAD

**Issue by osiloke** [lazywei/go-opencv#33]
I'm getting this strange bug where i can save the IplImage using opencv but when i try to do the same with the GoImage version, it creates a black image. I checked that the IplImage and GoImage pixels were similar using Get2D and At respectively. I need to ship the opencv image to another process so i need to get the raw data. The only way i could get this is using goimage, encoding to base64 sending it.

Basically,

buf := new(bytes.Buffer)
err := jpeg.Encode(buf, IplImage.ToImage(), nil)
bytestring_to_send := base64.StdEncoding.EncodeString(buf.Bytes())

**Comment by lazywei** [comment]
Hi, thanks to @paydro , this seems to be fixed in #40 
Please reopen if there is any concern.
Thanks.
